psi/app/__init__.py

```diff
@@ -85,11 +86,7 @@ def init_db(flask_app):
 
 
 def init_migrate(flask_app, database):
-    from flask_migrate import Migrate, upgrade
-
     Migrate(app=flask_app, db=database)
-    with flask_app.app_context():
-        upgrade(directory=os.path.dirname(__file__) + "/../migrations")
```

Comment:
Note: this hard-codes the `migrations/` directory as a subdirectory of the root (not `psi/`). I moved the migrations directory out of `psi/` to reflect this.

I think this worked before because you added the `psi/` directory to the $PATH manually.

Reply:
The reason behind why I move migration into psi is mainly because of two reasons

1. Logically to treat all the migrations as part of the system, they are necessary to make system work.
2. To reduce number of entries in the root folder.

I would prefer if we could fix the hard code directory but still keep the migrations folder under psi.

Any suggestion here?

Reply:
I can move it back in- I'll submit a change for that on this PR

Reply:
Ok this is done now

Reply:
Thanks :)